docs/cli/overview.mdx

```diff
@@ -81,12 +81,12 @@ The Infisical CLI provides a way to inject environment variables from the platfo
 		```bash
 		curl -1sLf \
 		'https://dl.cloudsmith.io/public/infisical/infisical-cli/setup.deb.sh' \
-		| sudo -E bash
+		| bash
 		```
 
     	Then install CLI
     	```bash
-    	sudo apt-get update && sudo apt-get install -y infisical
+    	apt-get update && apt-get install -y infisical
```

Comment:
sudo is needed to install/update via apt-get. I have tried on my ubuntu and it says permission denied without sudo